msmbuilder/libdistance/libdistance.pyx

```diff
@@ -356,6 +409,59 @@ cdef _assign_nearest_float(float[:, ::1] X, float[:, ::1] Y,
     return np.array(assignments, copy=False), inertia
 
 
+cdef _cdist_rmsd(XA, XB):
+    cdef npy_intp i, j
+    cdef float[:, :, ::1] XA_xyz = XA.xyz
+    cdef float[:, :, ::1] XB_xyz = XB.xyz
+    cdef npy_intp XA_length = XA_xyz.shape[0]
+    cdef npy_intp XB_length = XB_xyz.shape[0]
+    cdef int n_atoms = XA_xyz.shape[1]
+    cdef float[::1] XA_trace, XB_trace
+    cdef double[:, ::1] out
+
+    if XA._rmsd_traces is None:
+        raise ValueError('XA must be pre-centered using '
+                         'md.Trajectory.center_coordinates')
+    if XB._rmsd_traces is None:
+        raise ValueError('XB must be pre-centered using '
+                         'md.Trajectory.center_coordinates')
+
```

Comment:
@rmcgibbo instead of this error message can I tell it to do XA.center_coordinates()? e.g.

+    if XA._rmsd_traces is None:
 +        XA.center_coordinates()